### PR TITLE
Add `istio-system` namespace to module manifest

### PR DESCRIPTION
**Pull request by videlov**
**Description**

Changes proposed in this pull request:

- Add `istio-system` namespace to module manifest

**Pre-Merge Checklist**

- [ ] As a PR reviewer, verify code coverage and evaluate if it is acceptable.

**Related issues**

https://github.com/kyma-project/istio/issues/997

### CODE REVIEW

**Comment by videlov** [comment]
For the review, unfortunately additional namespace can not be applied with `kustomize` so we have to rely on bash application:
https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/namespace/

File: docs/release-notes/1.10.0.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ## New Features
 - Remove Istio version `v1alpha1` from the CRD [#1062](https://github.com/kyma-project/istio/pull/1062)
+- We create `istio-system` namespace as part of Module manifest[#1076](https://github.com/kyma-project/istio/pull/1076) following decision for [namespace labeling](https://github.com/kyma-project/community/issues/922)
```
Comment (werdes72) [inline docs/release-notes/1.10.0.md:3]:
```suggestion
- We create `istio-system` namespace as part of Module manifest [#1076](https://github.com/kyma-project/istio/pull/1076) following decision for [namespace labeling](https://github.com/kyma-project/community/issues/922)
```

Comment (nataliasitko) [inline docs/release-notes/1.10.0.md:3]:
```suggestion
- Now, the `istio-system` namespace is created as part of the module's manifest ([#1076](https://github.com/kyma-project/istio/pull/1076)). See the following [decision for namespace labeling](https://github.com/kyma-project/community/issues/922).
```

**Review by werdes72 — DISMISSED** [review]

**Review by nataliasitko — DISMISSED** [review]

File: config/namespace/istio_system_namespace.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: istio-system
+  labels:
+    operator.kyma-project.io/managed-by: kyma
```
Comment (kwiatekus) [inline config/namespace/istio_system_namespace.yaml:7]:
```suggestion
    operator.kyma-project.io/managed-by: kyma
    namespaces.warden.kyma-project.io/validate: enabled
```

**Review by werdes72 — APPROVED** [review]

**Review by nataliasitko — APPROVED** [review]

**Pull request merged**